libs/java/instance_provider/src/main/java/com/yahoo/athenz/instance/provider/impl/InstanceAzureProvider.java

```diff
@@ -138,40 +137,16 @@ public void initialize(String provider, String providerEndpoint, SSLContext sslC
         } catch (Exception ex) {
             LOGGER.error("Azure HTTP Client not created - no instance requests will be authorized");
             httpDriver = null;
-            enabled = false;
-        }
-
-        // if our settings are not valid there is no point of starting
-        // our timer thread to obtain AD access token
-
-        if (enabled) {
-
-            // let's first manually fetch the access token
-
-            try {
-                fetchAccessToken();
-            } catch (Exception ex) {
-                LOGGER.error("Unable to fetch VM access token", ex);
-            }
-
-            // now setup our credential updater
-
-            int credsUpdateTime = Integer.parseInt(System.getProperty(AZURE_PROP_TOKEN_UPDATE_TIMEOUT, "10"));
-
-            scheduledThreadPool = Executors.newScheduledThreadPool(1);
-            scheduledThreadPool.scheduleAtFixedRate(new AzureCredentialsUpdater(), credsUpdateTime,
-                    credsUpdateTime, TimeUnit.MINUTES);
         }
     }
 
     @Override
     public void close() {
+        /* For some reason, the close method is called after each confirmInstance, but the provider is reused, so we can't actually close it.
```

Comment:
It's because the InstanceAzureProvider used to be a HTTP-type provider, and those are created (and closed) for each request. I think I'll just remove the close here (the default method is empty).